dev-tools/mage/settings.go

```diff
@@ -721,6 +720,62 @@ func getProjectRepoInfoUnderGopath() (*ProjectRepoInfo, error) {
 	}, nil
 }
 
+var vcsList = []string{
+	"hg",
+	"git",
+	"svn",
+	"bzr",
+}
+
+func fromDir(dir, srcRoot string) (root string, err error) {
```

Comment:
[Question]
Is this an exact copy of `vcs.FromDir`, if so should we mention where it came from?